engine/src/main/java/com/datatorrent/stram/plan/physical/PhysicalPlan.java

```diff
@@ -1675,10 +1710,36 @@ public final void addLogicalOperator(OperatorMeta om)
     } else {
       initPartitioning(pnodes, 0);
     }
+    updateUpstreamsOutputMappings(om, null);
```

Comment:
Spelling. Also not clear why it is needed. Is it to fix an existing bug or is it for newly introduced functionality?